web-app/src/components/tournament/creator/teamBuilder.vue

```diff
@@ -0,0 +1,234 @@
+                  </q-item>
+
+                  <q-btn
+                    class="col-1"
+                    dense
+                    flat
+                    text-color="red"
+                    icon="close"
+                    @click="deletePlayer(item)"
+                  />
+                </q-item-label>
+              </q-item-section>
+            </q-item>
+          </template>
+        </q-virtual-scroll>
+      </q-card-section>
+      <q-card-section class="row items-stretch content-center q-pb-none">
+        <q-select
+          :class="$q.screen.gt.xs ? 'col-11 q-pb-none' : 'col-12'"
+          clearable
+          filled
+          use-input
+          v-model="player"
+          input-debounce="100"
+          :options="filterOptions"
+          :label="$t('tournament.team.choosePlayer')"
+          @filter="selectFilter"
+          @input="addPlayerControl"
+          emit-value
+          behavior="menu"
+          :error-message="playerSelectErrorMessage"
+          :error="isErrorSelectPlayer"
+        >
+          <template v-slot:selected-item>
+            <q-item v-if="player" class="q-pa-none">
+              <q-item-section avatar>
+                <q-avatar>
+                  <img :src="player.avatarSrc" />
+                </q-avatar>
+              </q-item-section>
+              <q-item-section>
+                {{ player.firstName }} {{ player.lastName }}
+              </q-item-section>
+            </q-item>
+          </template>
+          <template v-slot:no-option>
+            <q-item>
+              <q-item-section class="text-grey">
+                {{ $t('common.noSearchResults') }}
+              </q-item-section>
+            </q-item>
+          </template>
+          <template v-slot:option="scope">
+            <q-item v-bind="scope.itemProps" v-on="scope.itemEvents">
+              <q-item-section avatar>
+                <q-avatar>
+                  <img :src="scope.opt.avatarSrc" />
+                </q-avatar>
+              </q-item-section>
+              <q-item-section>
+                {{ scope.opt.firstName }} {{ scope.opt.lastName }}
+              </q-item-section>
+            </q-item>
+          </template>
+        </q-select>
+        <q-btn
+          v-if="$q.screen.gt.xs"
+          class="col-1"
+          text-color="white"
+          color="green"
+          icon="add"
+          dense
+          @click="addPlayer"
+        />
+      </q-card-section>
+      <q-card-section
+        class="q-pt-none"
+        :style="$q.screen.gt.xs ? 'position:absolute; bottom:0' : ''"
+      >
+        <q-btn
+          :disabled="players.length < 1"
+          :label="$t('tournament.team.add')"
+          type="submit"
+          color="primary"
+          class="q-mt-md"
+          :style="$q.screen.gt.xs ? '' : 'width:100%'"
+        >
+          <q-tooltip v-if="players.length < 1" content-class="bg-accent">
+            {{ $t('tournament.atLeastOneMemberError') }}
```

Comment:
```suggestion
            {{ $t('tournament.team.atLeastOneMemberError') }}
```
Does this error occur only in team